src/timeseriesflattener/flattened_dataset.py

```diff
@@ -450,26 +232,14 @@ def _get_temporal_feature(
         Returns:
             pd.DataFrame: Feature
         """
-        file_name = f"{feature_spec.get_col_str()}_{self.n_uuids}_uuids"
-
-        if hasattr(self, "feature_cache_dir"):
-            if self._cache_is_hit(
-                file_pattern=file_name,
-                output_spec=feature_spec,
-                file_suffix="parquet",
-            ):
-                df = self._load_cached_df_and_expand_fallback(
-                    file_pattern=file_name,
-                    full_col_str=feature_spec.get_col_str(),
-                    fallback=feature_spec.fallback,
-                    file_suffix=file_suffix,
-                )
 
-                return df.set_index(keys=self.pred_time_uuid_col_name).sort_index()
+        if self.cache and self.cache.feature_exists(feature_spec=feature_spec):
+            df = self.cache.read_feature(feature_spec=feature_spec)
+            return df.set_index(keys=self.pred_time_uuid_col_name).sort_index()
         else:
-            msg.info("No cache dir specified, not attempting load")
+            msg.info("No cache specified, not attempting load")
```

Comment:
Will print this even if the cache is specified but just hasn't been hit

Reply:
Should probably also be changed to a `debug` level message when we switch to using logging